packaging/fedora/snap-mgmt.sh

```diff
@@ -8,7 +8,20 @@
 set -e
 
 SNAP_MOUNT_DIR="/var/lib/snapd/snap"
-SNAP_UNIT_PREFIX="var-lib-snapd-snap"
+SNAP_UNIT_PREFIX="$(systemd-escape -p $SNAP_MOUNT_DIR)"
+
+show_help() {
+    exec cat <<'EOF'
+Usage: snap-mgmt.sh [OPTIONS]
+
+A simple script to cleanup snap installations.
+
+optional arguments:
+  --help                           Show this help message and exit
+  --snap-mount-dir=<path>          Provide a path to be used as $SNAP_MOUNT_DIR
+  --purge                          Purge all data from $SNAP_MOUNT_DIR
+EOF
+}
```

Comment:
Actually I don't see a difference if the script is called by a human or just another script. That it is not intended to be called by humans is already encoded by putting it no into `%_bindir` as you say. Providing a help output and proper command line options makes it just more convenient for developers to know what a script they call from another one does instead of relying on non obvious positional command line options. Hiding these things doesn't help in any case.